README.md

```diff
@@ -407,7 +400,7 @@ still can.
 ## Core technologies
 
 * Based on Redux + React
-* Code written in ES2015+
+* Code is written in ES2015+
```

Comment:
I don't see that this changes make an improvement. `Code is` sounds clunky. Maybe it should be `The code is written...`?